### ISSUE TITLE
Will be opportunity JSON view/edit in the DBeaver CE?

### ISSUE THREAD

**Issue by typo3ua** [dbeaver/dbeaver#10804]
I need to edit JSON just like that...

![image](https://user-images.githubusercontent.com/638022/103156724-6b9bf380-47b4-11eb-9aec-187b28bb9c86.png)

**Comment by kseniiaguzeeva** [comment]
You already can do it directly in cell, in cell editor or in Value panel https://github.com/dbeaver/dbeaver/wiki/Working-with-XML-and-JSON

**Comment by typo3ua** [comment]
Thank you for your reply!

i downloaded and test CE but do not found this opportunity there

it is EE version not CE... this is showing how can to do it in  EE version but not CE.

Can you give me link on a manual for CE version?

Thanks!

**Comment by kseniiaguzeeva** [comment]
It is available in CE version too. Have you tried it? Does something go wrong?

**Comment by typo3ua** [comment]
Maybe but i do not found it... 

OK! Let's do it step by step

### 1. Fist step - Run DBeaver CE
![0](https://user-images.githubusercontent.com/638022/103286193-822a9080-49e8-11eb-9be2-0e6ceb576efe.png)

### 2. Second step - select in the top menu "Open file..."
![1](https://user-images.githubusercontent.com/638022/103286428-c87fef80-49e8-11eb-9083-6f917b9e818a.png)

### 3. Third step - select JSON file
![2](https://user-images.githubusercontent.com/638022/103286530-fa915180-49e8-11eb-8c5a-2cd1e0766d3d.png)

### 4. Fourth step - open JSON file
![3](https://user-images.githubusercontent.com/638022/103286612-30ced100-49e9-11eb-8f4d-acdaf7f56f1e.png)

### ....that is all what i am find... what is next?

**Comment by uslss** [comment]
sorry for the late response.
@typo3ua do you mean the possibility to view and edit JSON documents as a usual table? Did I understand you correctly?

Btw, JSON view is available for non-SQL databases only (EE version). Values of JSON-like cells could be changed both in CE and EE versions.

**Comment by typo3ua** [comment]
Yes, possibility to view and edit JSON documents as a usual table... is it possible? .. I do not know how is it do it in the CE
